test/engine/test_pool.py

```diff
@@ -4,6 +4,8 @@
 import time
 import weakref
 
+import pytest
```

Comment:
**mike bayer** (zzzeek) wrote:

we can't import pytest in the tests themselves

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2373

Reply:
**Gord Thompson** (gordthompson) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2373